MediaBrowser.XbmcMetadata/Providers/SeasonFromSeriesNfoProvider.cs

```diff
@@ -63,6 +63,16 @@ protected override void Fetch(MetadataResult<Season> result, string path, Cancel
         /// <inheritdoc />
         protected override FileSystemMetadata? GetXmlFile(ItemInfo info, IDirectoryService directoryService)
         {
+            var seasonPath = info.Path;
+            if (seasonPath is not null)
+            {
+                var path = Path.Combine(seasonPath, "tvshow.nfo");
```

Comment:
isn't the season nfo file named differently?

Reply:
also, will this interfere with the normal season nfo parsing? or will the other season parser work independently from this parser?

Reply:
Season NFOs have their own provider and are parsed separately. I checked it and apparently we the returned list is sorted alphabetically, so I renamed the new provider to be run after the existing season provider.